packages/react-error-overlay/src/overlay.js

```diff
@@ -170,6 +170,7 @@ function crash(error: Error, unhandledRejection = false) {
 }
 
 function shortcutHandler(type: string) {
+  if (errorReferences.length === 0) return;
```

Comment:
Please move this to `switchError(offset)`, and make sure the return is within brackets, e.g.:

```js
function switchError(offset) {
  if (errorReferences.length < 1) {
    return;
  }
  // ...
}
```